internal/postgres/utils.go

```diff
@@ -20,7 +20,71 @@ type PgxArray[T any] struct {
 // properly handles scanning postgres arrays
 func (a *PgxArray[T]) Scan(src any) error {
 	m := pgtype.NewMap()
-	pgt, ok := m.TypeForName(strings.ToLower(a.colDataType))
+	// Register money types
+	m.RegisterType(&pgtype.Type{
+		Name:  "money",
+		OID:   790,
+		Codec: pgtype.TextCodec{},
+	})
+	m.RegisterType(&pgtype.Type{
+		Name: "_money",
+		OID:  791,
+		Codec: &pgtype.ArrayCodec{
+			ElementType: &pgtype.Type{
+				Name:  "money",
+				OID:   790,
+				Codec: pgtype.TextCodec{},
+			},
+		},
+	})
+
+	// Register UUID types
+	m.RegisterType(&pgtype.Type{
```

Comment:
Why do you have to register the uuid type when pgx has a `pgtype.UUID`?

Reply:
I'm overriding the codec so that is returns the uuid as a string instead of []byte

Reply:
makes sense

Reply:
kinda surprised youre able to register it since pgx already registers the uuid type, unless it just overrides it (which I'm guessing is what happens)

Reply:
yep its just a map so the value for that key gets updated when you register